tools/msvs/vswhere_usability_wrapper.cmd

```diff
@@ -1,20 +1,32 @@
 :: Copyright 2017 - Refael Ackermann
 :: Distributed under MIT style license
 :: See accompanying file LICENSE at https://github.com/node4good/windows-autoconf
-:: version: 1.14.0
+:: version: 1.15.4
 
 @if not defined DEBUG_HELPER @ECHO OFF
 setlocal
+set "InstallerPath=%ProgramFiles(x86)%\Microsoft Visual Studio\Installer"
+if not exist "%InstallerPath%" set "InstallerPath=%ProgramFiles%\Microsoft Visual Studio\Installer"
+if not exist "%InstallerPath%" goto :no-vswhere
+:: Manipulate %Path% for easier " handeling
+set Path=%Path%;%InstallerPath%
```

Comment:
Actually I can't remember why I use the `%Path%` trick... But I do remember that `"%InstallerPath%"\vswhere.exe` doesn't work is some cases